Custom select validation tweaks

MartijnCuppens opened the pull request:
The padding of custom selects doesn't cover the background, if the text in the select box is to long, the text and background image will overlap.

 This PR also removes unneeded `calc()` functions.

Before:
![image](https://user-images.githubusercontent.com/11559216/44999992-94839380-afc1-11e8-94fb-79435b40c7bf.png)

After:
![image](https://user-images.githubusercontent.com/11559216/44999921-05767b80-afc1-11e8-878c-95aca9c2b561.png)

mdo commented on scss/mixins/_forms.scss:
```diff
@@ -98,12 +98,12 @@
 
         @if $state == "valid" {
-          background: $custom-select-background, $form-feedback-icon-valid no-repeat center right ($input-height-inner * .9) / calc(#{$input-height-inner} / 2) calc(#{$input-height-inner} / 2);
+          background: $custom-select-background, $form-feedback-icon-valid no-repeat center right ($custom-select-padding-x + $custom-select-indicator-padding) unquote("/") ($input-height-inner / 2) ($input-height-inner / 2);
```
I wonder if we take the opportunity to clean this up a bit here and turn these major values into variables? Separately, was the `/` working as intended before? Curious if Sass was trying to compute this vs keeping it as a property separator.

MartijnCuppens replied:
A `calc()` function was used, that's why the `/` worked before.

Do you want to use the variables in the `_variables.scss` file or declare them here in the `_forms.scss`?

mdo replied:
Ah, yeah, got it. Most likely in `_variables.scss`.

XhmikosR commented:
@MartijnCuppens: can you move the variable to _variables.scss?

MartijnCuppens committed:
Custom select validation padding fix and additional custom select feedback icon variables

MartijnCuppens commented:
I've moved some values to the variables file as mdo suggested and simplified the `if/else` construction. Also fixed the linting issue and squashed everything into one commit.

XhmikosR committed:
Merge branch 'v4-dev' into custom-select-validation-padding

XhmikosR committed:
Merge branch 'v4-dev' into custom-select-validation-padding
